src/commands.rs

```diff
@@ -64,6 +66,10 @@ pub enum CellarRebalancerCmd {
 
     #[options(help = "remove_funds")]
     RemoveFunds(RemoveFundsCmd),
+
+    // Gorc subcommand from gravity-bridge
+    #[options(help = "gorc sub-commands")]
+    Orchestrator(OrchestratorCmd),
```

Comment:
For now, would it be possible to make `OrchestratorCmd` a thin wrapper around something shared in another crate, and copy-and-paste its definition into this application?

Otherwise we can look at trying to make it generic around the underlying Abscissa application, but that's a lot more work.